net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml

```diff
@@ -3,6 +3,7 @@
     <General VisibleName="General" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/general/index" order="1"/>
     <RIP VisibleName="RIP" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/rip/index" order="10" />
     <OSPF VisibleName="OSPF" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf/index" order="20" />
+    <OSPF6 VisibleName="OSPFv3" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf6/index" order="25" />
```

Comment:
@fichtner this comes from me from testing (just having some icon and there is no other idea behind it). If you are having a good idea which icon characterises link state, distance vector etc. it would be cool. I use the way sign for routing (global menu) already.

Maybe use use those:
OSPF (all variants): http://fontawesome.io/icon/map/
RIP (all variants): http://fontawesome.io/icon/expand/
BGP: http://fontawesome.io/icon/globe/